src/exoscraper/target.py

```diff
@@ -37,7 +37,49 @@ def lightcurve(self):
         # go get the TESS data something like
         # return get_timeseries(self.ra, self.dec)
         raise NotImplementedError
+
+    @property
+    def bibliography(self):
+        # go get references from NASA ADS
+        # return get_bibliography(self.name)
+        raise NotImplementedError
+
+    @property
+    def transit_times(self, time: u.Quantity):
+        # calculate future transit times
+        # return array of future transit times out to a specified time
+        raise NotImplementedError
+
+    @property
+    def SED(self):
+        # calculate SED of target star
+        # return idk whatever Christina has written
+        raise NotImplementedError
+
+    @property
+    def transit_model(self) -> lk.LightCurve:
+        # generate transit model using lightkurve
+        # return LightCurve object
+        raise NotImplementedError
+
+    @property
+    def noise_model(self, mission: str) -> lk.LightCurve:
```

Comment:
This is going to need an input light curve (what times do you want it estimated at) or it needs to output a function that acts on a light curve.